src/utilsAPI.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
```

Comment:
Consider setting up a base URL using axios.create, this makes it easier to reuse 👍